mixer/pkg/server/monitoring.go

```diff
@@ -61,6 +62,14 @@ func startMonitor(port uint16) (*monitor, error) {
 		}
 	})
 
+	if enableProfiling {
```

Comment:
Can you create a unit test for this? The server package has 100% coverage, it'd be nice to stay there :-)

Reply:
the default being `true`, it's covered